static/js/stream_popover.js

```diff
@@ -402,7 +402,7 @@ exports.register_stream_handlers = function () {
     $("body").on("click", ".toggle_home", (e) => {
```

Comment:
We might want to rename this `toggle_home` in an extra commit while you're at it.

Reply:
To like `toggle_mute`?

Reply:
Actually I’ll go with `toggle_stream_muted` to avoid confusion with our `toggle_mute` for messages.